meta-balena-common/recipes-containers/balena-supervisor/balena-supervisor/update-balena-supervisor

```diff
@@ -166,11 +166,7 @@ fi
 
 # Pull target version.
 echo "Pulling supervisor $image_name:$tag..."
-if $DOCKER pull "$image_name:$tag"; then
-    $DOCKER rm --force balena_supervisor || true
-else
-    error_handler "supervisor pull failed"
-fi
+$DOCKER pull "$image_name:$tag" || error_handler "supervisor pull failed"
```

Comment:
Avoid force removing the existing supervisor container here as we may have set `--no-start-stop-supervisor`.

Reply:
We may want to tag the image here to allow later removal, LMK

```bash
$DOCKER tag "${image_id}" "${SUPERVISOR_NAME}:${version}"
```